Library/Homebrew/install.rb

```diff
@@ -28,7 +28,7 @@ def attempt_directory_creation
           # Create these files to ensure that these directories aren't removed
           # by the Catalina installer.
           # (https://github.com/Homebrew/brew/issues/6263)
-          keep_file = dir/".keepme"
+          keep_file = DevelopmentTools.keep_file(dir)
```

Comment:
```suggestion
          keep_file = dir/".keepme"
```

Reply:
Thanks @MikeMcQuaid for the review. I've updated the PR with the requested changes.

There was a `to_s` missing, which I've added in https://github.com/Homebrew/brew/commit/51264f15e08f809c9772d909dc11d01745196a4f#diff-0c1a9253e1740bb55c4d5db123daad9dR429